src/program/services/post_processing/subtitles/providers/opensubtitles_com.py

```
@@ -0,0 +1,461 @@
+            logger.trace(f"Trying search strategy: {strategy['name']} with params={strategy['params']}")
+            results = self._search(strategy["params"])
+            if results:
+                return self._score_results(results, strategy["name"])
+
+        logger.debug(f"No subtitles found for language={language}")
+        return []
+
+    def _build_search_strategies(
+        self,
+        video_hash: str | None,
+        file_size: int | None,
```

Ruff (0.14.14)

[warning] Unused method argument: `file_size`

(ARG002)